## OPENPEDCAN METHODS

### RESOURCE AVAILABILITY

#### Lead contact

Requests for access to OpenPedCan raw data and/or specimens may be directed to, and will be fulfilled by Jo Lynne Rokita (rokita@chop.edu).

#### Materials availability

This study did not create new, unique reagents.

#### Data and code availability

Within OpenPedCan (OPC), we harmonized, aggregated, and analyzed data from multiple sources. 
We harmonized data from the Therapeutically Applicable Research to Generate Effective Treatments (TARGET cohort) Initiative, an NCI-funded collection of disease-specific projects that seeks to identify the genomic changes of pediatric cancers [@dbgap:phs000218.v23.p8]. 
We included already harmonized neuroblastoma samples from the Gabriella Miller Kids First (GMKF cohort) Pediatric Research Program, a large-scale effort to accelerate research and gene discovery in pediatric cancers and structural birth defects [@dbgap:phs001436.v1.p1].
Additionally, we re-harmonized all samples from the Open Pediatric Brain Tumor Atlas (OpenPBTA, PBTA cohort), an open science initiative led by Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab and the Center for Data-Driven Discovery (D3B) at the Children’s Hospital of Philadelphia (CHOP), which genomically characterized pediatric brain tumor data from the Children’s Brain Tumor Network (CBTN), and the Pacific Pediatric Neuro-oncology Consortium (PNOC) [@dbgap:phs002517.v1.p1; @doi:10.1016/j.xgen.2023.100340].
Building on the work of OpenPBTA, OPC added the PBTA X01 data [@https://commonfund.nih.gov/kidsfirst/2021X01projects#FY21_Resnick], the Chordoma Foundation data [@https://www.chordomafoundation.org/], and the MI-ONCOSEQ Study [@https://www.pathology.med.umich.edu/mctp/mi-oncoseq-study], donated to CBTN by the University of Michigan, to the PBTA cohort.
Finally, OPC includes the Children's Hospital of Philadelphia (CHOP) P30 Panel data generated by CHOP's Division of Genomic Diagnostics (DGD cohort) which includes fusion panel data [@dbgap:phs002517.v1.p1].
In addition to pediatric cancer data, OpenPedCan contains adult data from large science consortiums as references.
For normal gene expression, GTEx [@dbgap:phs000424.v9.p2] was used, and for comparison with adult cancers, The Cancer Genome Atlas (TCGA) [@https://www.cancer.gov/tcga] was included.

Merged summary files for OpenPedCan v12 are openly accessible in [CAVATICA](https://cavatica.sbgenomics.com/u/cavatica/opentarget) or via `download-data.sh` script in the [https://github.com/PediatricOpenTargets/OpenPedCan-analysis](https://github.com/PediatricOpenTargets/OpenPedCan-analysis) repository.
Cancer group summary data are visible within the NCI's pediatric [Molecular Targets Platform](https://moleculartargets.ccdi.cancer.gov/) and cohort, cancer group, and individual data are visible within [PedcBioPortal](https://pedcbioportal.kidsfirstdrc.org/study/summary?id=openpedcan_v12)

OpenPedCan analysis modules were developed within OpenPBTA [@doi:10.1016/j.xgen.2023.100340], modified based on OpenPBTA, or newly created and can be found within the following publicly available repositories.
OpenPBTA module analyses can be found at [https://github.com/AlexsLemonade/OpenPBTA-analysis](https://github.com/AlexsLemonade/OpenPBTA-analysis).
OpenPedCan module analyses can be found at [https://github.com/PediatricOpenTargets/OpenPedCan-analysis](https://github.com/PediatricOpenTargets/OpenPedCan-analysis).
OpenPedCan api code can be found at [https://github.com/PediatricOpenTargets/OpenPedCan-api](https://github.com/PediatricOpenTargets/OpenPedCan-api).

All original code was developed within the following modules in the OpenPedCan analyses repository as listed below. Links to the modules are available here, and within each module is a detailed README that describes the purpose and intended usage of the scripts, along with pointers to the results from the data those scripts process. 

[chromosomal-instability](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/chromosomal-instability)
[cnv-frequencies](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/cnv-frequencies)
[collapse-rnaseq](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/collapse-rnaseq)
[compare-gistic](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/compare-gistic)
[copy_number_consensus_call](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/copy_number_consensus_call)
[create-subset-files](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/create-subset-files)
[data-pre-release-qc](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/data-pre-release-qc)
[efo-mondo-mapping](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/efo-mondo-mapping)
[filter-mtp-tables](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/filter-mtp-tables)
[focal-cn-file-preparation](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/focal-cn-file-preparation)
[fusion-frequencies](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/fusion-frequencies)
[fusion-summary](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/fusion-summary)
[fusion_filtering](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/fusion_filtering)
[gene-set-enrichment-analysis](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/gene-set-enrichment-analysis)
[gene_match](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/gene_match)
[immune-deconv](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/immune-deconv)
[independent-samples](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/independent-samples)
[long-format-table-utils](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/long-format-table-utils)
[methylation-preprocessing](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/methylation-preprocessing)
[methylation-summary](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/methylation-summary)
[molecular-subtyping-ATRT](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/molecular-subtyping-ATRT)
[molecular-subtyping-CRANIO](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/molecular-subtyping-CRANIO)
[molecular-subtyping-EPN](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/molecular-subtyping-EPN)
[molecular-subtyping-EWS](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/molecular-subtyping-EWS)
[molecular-subtyping-HGG](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/molecular-subtyping-HGG)
[molecular-subtyping-LGAT](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/molecular-subtyping-LGAT)
[molecular-subtyping-MB](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/molecular-subtyping-MB)
[molecular-subtyping-NBL](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/molecular-subtyping-NBL)
[molecular-subtyping-chordoma](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/molecular-subtyping-chordoma)
[molecular-subtyping-embryonal](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/molecular-subtyping-embryonal)
[molecular-subtyping-integrate](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/molecular-subtyping-integrate)
[molecular-subtyping-neurocytoma](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/molecular-subtyping-neurocytoma)
[molecular-subtyping-pathology](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/molecular-subtyping-pathology)
[mtp-annotations](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/mtp-annotations)
[mtp-tables-qc-checks](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/mtp-tables-qc-checks)
[mutational-signatures](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/mutational-signatures)
[pedcbio-cnv-prepare](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/pedcbio-cnv-prepare)
[pedcbio-sample-name](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/pedcbio-sample-name)
[pedot-table-column-display-order-name](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/pedot-table-column-display-order-name)
[rna-seq-expression-summary-stats](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/rna-seq-expression-summary-stats)
[rnaseq-batch-correct](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/rnaseq-batch-correct)
[run-gistic](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/run-gistic)
[snv-frequencies](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/snv-frequencies)
[tmb-calculation](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/tmb-calculation)
[tp53_nf1_score](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/tp53_nf1_score)
[tumor-gtex-plots](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/tumor-gtex-plots)
[tumor-normal-differential-expression](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/tumor-normal-differential-expression)


Software versions are documented in **Table XX**.

#### Data releases
We maintained a data release folder on Amazon S3, downloadable directly from S3 or our open-access CAVATICA project, with merged files for each analysis (See data and code availability section). 
As we produced new results that we expected to be used across multiple analyses, or identified data issues, we created new data releases in a versioned manner. 

### EXPERIMENTAL MODEL AND STUDY PARTICIPANT DETAILS
<!-- TODO: add description of all studies here -->



### METHOD DETAILS

#### Nucleic acids extraction and library preparation
<!-- TODO: add by study here, or refer to publication -->


#### Data generation
<!-- TODO: add by study here, or refer to publication -->


#### DNA WGS Alignment
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].

#### Quality Control of Sequencing Data
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].


##### SNP calling for B-allele Frequency (BAF) generation
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].


#### Somatic Mutation Calling

##### SNV and indel calling
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].


##### VCF annotation and MAF creation
<!-- TODO: update VEP version -->


##### Gather SNV and INDEL Hotspots
<!-- TODO: needs update -->


##### Consensus SNV Calling
<!-- TODO: needs update -->


#### Somatic Copy Number Variant Calling (WGS samples only)
<!-- TODO: needs update -->

##### Consensus CNV Calling
<!-- TODO: needs update -->


#### Somatic Structural Variant Calling (WGS samples only)
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].


#### Methylation Analysis
##### Methylation array preprocessing
We preprocessed raw Illumina 450K and EPIC 850K Infinium Human Methylation Bead Array intensities using the array preprocessing methods implemented in the `minfi Bioconductor package` [@doi:10.1093/bioinformatics/btw691]. 
We utilized either `preprocessFunnorm` when an array dataset had both tumor and normal samples or multiple OpenPedcan-defined `cancer_groups` and `preprocessQuantile` when an array dataset had only tumor samples from a single OpenPedcan-defined `cancer_group` to estimate usable methylation measurements (`beta-values` and `m-values`) and copy number (`cn-values`). 
Some Illumina Infinium array probes targeting CpG loci contain single-nucleotide polymorphisms (SNPs) near or within the probe [@doi:10.1038/bjc.2013.496], which could affect DNA methylation measurements [@doi:10.1186/s12864-015-2202-0]. 
As the minfi preprocessing workflow recommends, we dropped probes containing common SNPs in dbSNP (minor allele frequency > 1%) at the CpG interrogation or the single nucleotide extensions.

Details of methylation array preprocessing are available in the [OpenPedCan methylation-preprocessing module](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/methylation-preprocessing).

##### Methylation beta-values summaries
We comprehensively summarized gene-level and isoform-level metrics for the methylation `beta-values` estimated by array preprocessing to provide insight into the variations in overall genomic DNA methylation levels observed across different pediatric tumors by computing CpG probe-level summary metrics in each cancer group within a cohort, including 1) `beta-values quantiles`, 2) `gene expression (TPM) and methylation (beta-values) correlation`, 3) `TPM median expression`, and 4) `transcript representation` - a proxy for percent isoform expression in a gene. 
In addition, each CpG probe was annotated with a gene feature to identify the genomic regions likely involved in regulating gene expression. 

Details of the analysis are available in the [OpenPedCan methylation-summary module](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/methylation-summary).

##### Methylation sample classification
We ran the [dkfz's brain classifier version 12.5](https://www.molecularneuropathology.org/mnp/classifiers/11), a comprehensive DNA methylation-based classification of CNS tumors across all entities and age groups [@doi:10.1038/nature26000]. 
Unprocessed IDAT-files from the [Children's Brain Tumor Network (CBTN)](https://cbtn.org/) `Infinium Human Methylation EPIC (850k) BeadChip arrays` were used as input and the following information was compiled into the `histologies.tsv` file: `dkfz_v12_methylation_subclass` (predicted methylation subtype), `dkfz_v12_methylation_subclass_score` (classification score), `dkfz_v12_methylation_mgmt_status` (_MGMT_ methylation status), and `dkfz_v12_methylation_mgmt_estimated` (estimated _MGMT_ methylation fraction).



#### Gene Expression

##### Abundance Estimation
<!-- TODO: needs update -->


##### Gene Expression Matrices with Unique HUGO Symbols
<!-- TODO: needs update, include liftover for TCGA/GTEX -->


##### Gene Expression Summary Statistics
We generated RNA-Seq gene expression (TPM) summary statistics for independent tumor samples from the combined OpenPedCan gene expression matrices, including cancers from pediatric cohorts (`PBTA`, `GMKF`, and `TARGET`) and adult cancers from the `TCGA` cohort.
We grouped selected samples into two groups containing samples from a cancer group in either each cohort or all cohorts, and calculated `TPM means`, `standard deviations`, `gene-wise z-scores`, `group-wise z-scores`, and `ranks` for each group as described in the [OpenPedCan rna-seq-expression-summary-stats module](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/rna-seq-expression-summary-stats) in detail. 
The resulting `gene-wise` and `group-wise` summary statistics tables were annotated with `EFO` and `MONDO` disease codes associated with the cancer groups.


##### Gene fusion detection
<!-- TODO: needs update -->



### QUANTIFICATION AND STATISTICAL ANALYSIS

##### Focal Copy Number Calling (`focal-cn-file-preparation` analysis module)
Please refer to the OpenPBTA manuscript for details on assignment of copy number status values to CNV segments, cytobands, and genes [@doi:10.1016/j.xgen.2023.100340]. 
We applied criteria to resolve instances of multiple conflicting status calls for the same gene and sample, which are described in detail in the [focal-cn-file-preparation](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/focal-cn-file-preparation) module. 
Briefly, we prioritized 1) non-neutral status calls, 2) calls made from dominant segments with respect to gene overlap, and 3) amplification and deep deletion status calls over gain and loss calls, respectively, when selecting a dominant status call per gene and sample. 
These methods resolved >99% of duplicated gene-level status calls.    


##### Gene Set Variation Analysis (`gene-set-enrichment-analysis` analysis module)
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].



##### Fusion prioritization (`fusion_filtering` analysis module)




#### Mutational Signatures (`mutational-signatures` analysis module)



### Tumor Mutation Burden (`snv-callers` analysis module)



#### Clinical Data Harmonization

##### WHO Classification of Disease Types


##### Molecular Subtyping
Here, we build upon the molecular subtyping performed in OpenPBTA [@doi:10.1016/j.xgen.2023.100340].

High-grade gliomas..
<!-- TODO: needs update - DHG, IHG -->

Atypical teratoid rhabdoid tumors..


Neuroblastoma tumors...


###### Integration of brain tumor methylation classifications 


#### TP53 Alteration Annotation (`tp53_nf1_score` analysis module)
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].


#### Prediction of participants' genetic sex
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].


#### Selection of independent samples (`independent-samples` analysis module)

For analyses that require all input biospecimens to be independent, we use the OpenPedCan-analysis [independent-samples](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/d397339d567ddeff17e7a8cdca892f6a9dd2a0ba/analyses/independent-samples) module to select only one biospecimen from each input participant.
For each input participant of an analysis, the independent biospecimen is selected based on the analysis-specific filters and preferences for the biospecimen metadata, such as experimental strategy, cancer group, and tumor descriptor.

