## OPENPEDCAN METHODS

### RESOURCE AVAILABILITY

#### Lead contact

Requests for access to OpenPedCan raw data and/or specimens may be directed to, and will be fulfilled by Jo Lynne Rokita (rokita@chop.edu).

#### Materials availability

This study did not create new, unique reagents.

#### Data and code availability

Merged summary files for OpenPedCan v12 are openly accessible in [CAVATICA](https://cavatica.sbgenomics.com/u/cavatica/opentarget) or via download script in the [https://github.com/PediatricOpenTargets/OpenPedCan-analysis](https://github.com/PediatricOpenTargets/OpenPedCan-analysis) repository.
Cancer group summary data are visible within the NCI's pediatric [Molecular Targets Platform](https://moleculartargets.ccdi.cancer.gov/) and cohort, cancer group, and individual data are visible within [PedcBioPortal](https://pedcbioportal.kidsfirstdrc.org/study/summary?id=openpedcan_v12)

OpenPedCan analysis modules were developed within OpenPBTA [@doi:10.1016/j.xgen.2023.100340], modified based on OpenPBTA, or newly created and can be found within the following publicly available repositories.
OpenPBTA module analyses can be found at [https://github.com/AlexsLemonade/OpenPBTA-analysis](https://github.com/AlexsLemonade/OpenPBTA-analysis).
OpenPedCan module analyses can be found at [https://github.com/PediatricOpenTargets/OpenPedCan-analysis](https://github.com/PediatricOpenTargets/OpenPedCan-analysis).
OpenPedCan api code can be found at [https://github.com/PediatricOpenTargets/OpenPedCan-api](https://github.com/PediatricOpenTargets/OpenPedCan-api).

Software versions are documented in **Table XX**.

#### Data releases


### EXPERIMENTAL MODEL AND STUDY PARTICIPANT DETAILS
<!-- TODO: add description of all studies here -->



### METHOD DETAILS

#### Nucleic acids extraction and library preparation
<!-- TODO: add by study here, or refer to publication -->


#### Data generation
<!-- TODO: add by study here, or refer to publication -->


#### DNA WGS Alignment
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].

#### Quality Control of Sequencing Data
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].


##### SNP calling for B-allele Frequency (BAF) generation
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].


#### Somatic Mutation Calling

##### SNV and indel calling
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].


##### VCF annotation and MAF creation
<!-- TODO: update VEP version -->


##### Gather SNV and INDEL Hotspots
<!-- TODO: needs update -->


##### Consensus SNV Calling
<!-- TODO: needs update -->


#### Somatic Copy Number Variant Calling (WGS samples only)
<!-- TODO: needs update -->

##### Consensus CNV Calling
<!-- TODO: needs update -->

We adopted the consensus CNV calling described in OpenPBTA manuscript [doi:10.1016/j.xgen.2023.100340] with minor adjustments. 
For each caller and sample, we called CNVs based on consensus among Control-FREEC ([@https://doi.org/10.1093/bioinformatics/btq635]; [@https://doi.org/10.1093/bioinformatics/btr670]), CNVkit ([@doi: 10.1371/journal.pcbi.1004873]), and GATK ([@doi:
10.1101/gr.107524.110]). 
Sample and consensus caller files with more than 2,500 CNVs were removed to de-noise and increase data quality, based on cutoffs used in GISTIC ([@https://doi.org/10.1186/gb-2011-12-4-r41]). 
For each sample, we included the regions in the final consensus set: 1) regions with reciprocal overlap of 50% or more between at least two of the callers; 2) smaller CNV regions in which more than 90% of regions are covered by another caller. 
For GATK, if a panel of normal was not provided, the correlated sample would be skipped. 
All the copy number alteration included in the consensus file were shown in algorithm two algorithms. 
We defined copy number as NA for any regions that had a neutral call for the samples included in the consensus file. 
We merged CNV regions within 10,000 bp of each other with the same direction of gain or loss into single region. 
Any CNVs that overlapped 50% or more with immunoglobulin, telomeric, centromeric, segment duplicated regions, or that were shorter than 3000 bp were filtered out.


#### Somatic Structural Variant Calling (WGS samples only)
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].


#### Methylation Analysis
<!-- include QC, b-value/m-value calculations, packages, dkfz classification -->


#### Gene Expression

##### Abundance Estimation
<!-- TODO: needs update -->


##### Gene Expression Matrices with Unique HUGO Symbols
<!-- TODO: needs update, include liftover for TCGA/GTEX -->


##### Gene fusion detection
<!-- TODO: needs update -->


### QUANTIFICATION AND STATISTICAL ANALYSIS

##### Focal Copy Number Calling (`focal-cn-file-preparation` analysis module)



##### Gene Set Variation Analysis (`gene-set-enrichment-analysis` analysis module)
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].



##### Fusion prioritization (`fusion_filtering` analysis module)




#### Mutational Signatures (`mutational-signatures` analysis module)



### Tumor Mutation Burden (`snv-callers` analysis module)



#### Clinical Data Harmonization

##### WHO Classification of Disease Types


##### Molecular Subtyping
Here, we build upon the molecular subtyping performed in OpenPBTA [@doi:10.1016/j.xgen.2023.100340].

High-grade gliomas..
<!-- TODO: needs update - DHG, IHG -->

Atypical teratoid rhabdoid tumors..


Neuroblastoma tumors...


###### Integration of brain tumor methylation classifications 


#### TP53 Alteration Annotation (`tp53_nf1_score` analysis module)
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].


#### Prediction of participants' genetic sex
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].


#### Selection of independent samples (`independent-samples` analysis module)



