## OPENPEDCAN METHODS

### RESOURCE AVAILABILITY

#### Lead contact

Requests for access to OpenPedCan raw data and/or specimens may be directed to, and will be fulfilled by Jo Lynne Rokita (rokita@chop.edu).

#### Materials availability

This study did not create new, unique reagents.

#### Data and code availability

Merged summary files for OpenPedCan v12 are openly accessible in [CAVATICA](https://cavatica.sbgenomics.com/u/cavatica/opentarget) or via download script in the [https://github.com/PediatricOpenTargets/OpenPedCan-analysis](https://github.com/PediatricOpenTargets/OpenPedCan-analysis) repository.
Cancer group summary data are visible within the NCI's pediatric [Molecular Targets Platform](https://moleculartargets.ccdi.cancer.gov/) and cohort, cancer group, and individual data are visible within [PedcBioPortal](https://pedcbioportal.kidsfirstdrc.org/study/summary?id=openpedcan_v12)

OpenPedCan analysis modules were developed within OpenPBTA [@doi:10.1016/j.xgen.2023.100340], modified based on OpenPBTA, or newly created and can be found within the following publicly available repositories.
OpenPBTA module analyses can be found at [https://github.com/AlexsLemonade/OpenPBTA-analysis](https://github.com/AlexsLemonade/OpenPBTA-analysis).
OpenPedCan module analyses can be found at [https://github.com/PediatricOpenTargets/OpenPedCan-analysis](https://github.com/PediatricOpenTargets/OpenPedCan-analysis).
OpenPedCan api code can be found at [https://github.com/PediatricOpenTargets/OpenPedCan-api](https://github.com/PediatricOpenTargets/OpenPedCan-api).

Software versions are documented in **Table XX**.

#### Data releases


### EXPERIMENTAL MODEL AND STUDY PARTICIPANT DETAILS
<!-- TODO: add description of all studies here -->



### METHOD DETAILS

#### Nucleic acids extraction and library preparation
<!-- TODO: add by study here, or refer to publication -->


#### Data generation
<!-- TODO: add by study here, or refer to publication -->


#### DNA WGS Alignment
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].

#### Quality Control of Sequencing Data
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].


##### SNP calling for B-allele Frequency (BAF) generation
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].


#### Somatic Mutation Calling

##### SNV and indel calling
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].


##### VCF annotation and MAF creation
<!-- TODO: update VEP version -->


##### Gather SNV and INDEL Hotspots
<!-- TODO: needs update -->


##### Consensus SNV Calling
<!-- TODO: needs update -->


#### Somatic Copy Number Variant Calling (WGS samples only)
<!-- TODO: needs update -->

##### Consensus CNV Calling
<!-- TODO: needs update -->


#### Somatic Structural Variant Calling (WGS samples only)
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].


#### Methylation Analysis
##### Methylation array preprocessing
We preprocessed raw Illumina 450K and EPIC 850K Infinium Human Methylation Bead Array intensities using the array preprocessing methods implemented in the minfi Bioconductor package[@doi:10.1093/bioinformatics/btw691]. 
We utilized either `preprocessFunnorm` when an array dataset had both tumor and normal samples or multiple OpenPedcan-defined `cancer_groups` and `preprocessQuantile` when an array dataset had only tumor samples from a single OpenPedcan-defined `cancer_group` to estimate usable methylation measurements (`beta-values` and `m-values`) and copy number (`cn-values`). 
We dropped array probes containing SNPs at either the CpG interrogation or the single nucleotide extensions.

Details of methylation array preprocessing are available in the [OpenPedCan methylation-preprocessing module](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/methylation-preprocessing).

##### Methylation beta-values summaries
We comprehensively summarized gene-level and isoform-level metrics for the methylation `beta-values` estimated by array preprocessing to provide insight into the variations in overall genomic DNA methylation levels observed across different pediatric tumors by computing CpG probe-level summary metrics in each cancer group within a cohort, including 1) `beta-values quantiles`, 2) `gene expression (TPM) and methylation (beta-values) correlation`, 3) `TPM median expression`, and 4) `transcript representation` - a proxy for percent isoform expression in a gene. 
In addition, each CpG probe was annotated with a gene feature to identify the genomic regions likely involved in regulating gene expression. 

Details of the analysis are available in the [OpenPedCan methylation-summary module](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/methylation-summary).

##### Methylation sample classification
We ran the [dkfz's brain classifier version 12.5](https://www.molecularneuropathology.org/mnp/classifiers/11), a comprehensive DNA methylation-based classification of CNS tumors across all entities and age groups [@doi:10.1038/nature26000]. 
Unprocessed IDAT-files from the [Children's Brain Tumor Network (CBTN)](https://cbtn.org/) `Infinium Human Methylation EPIC (850k) BeadChip arrays` were used as input and the following information was compiled into the `histologies.tsv` file: `dkfz_v12_methylation_subclass` (predicted methylation subtype), `dkfz_v12_methylation_subclass_score` (classification score), `dkfz_v12_methylation_mgmt_status` (_MGMT_ methylation status), and `dkfz_v12_methylation_mgmt_estimated` (estimated _MGMT_ methylation fraction).



#### Gene Expression

##### Abundance Estimation
<!-- TODO: needs update -->


##### Gene Expression Matrices with Unique HUGO Symbols
<!-- TODO: needs update, include liftover for TCGA/GTEX -->


##### Gene fusion detection
<!-- TODO: needs update -->


### QUANTIFICATION AND STATISTICAL ANALYSIS

##### Focal Copy Number Calling (`focal-cn-file-preparation` analysis module)



##### Gene Set Variation Analysis (`gene-set-enrichment-analysis` analysis module)
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].



##### Fusion prioritization (`fusion_filtering` analysis module)




#### Mutational Signatures (`mutational-signatures` analysis module)



### Tumor Mutation Burden (`snv-callers` analysis module)



#### Clinical Data Harmonization

##### WHO Classification of Disease Types


##### Molecular Subtyping
Here, we build upon the molecular subtyping performed in OpenPBTA [@doi:10.1016/j.xgen.2023.100340].

High-grade gliomas..
<!-- TODO: needs update - DHG, IHG -->

Atypical teratoid rhabdoid tumors..


Neuroblastoma tumors...


###### Integration of brain tumor methylation classifications 


#### TP53 Alteration Annotation (`tp53_nf1_score` analysis module)
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].


#### Prediction of participants' genetic sex
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].


#### Selection of independent samples (`independent-samples` analysis module)



