## OPENPEDCAN METHODS

### RESOURCE AVAILABILITY

#### Lead contact

Requests for access to OpenPedCan raw data and/or specimens may be directed to, and will be fulfilled by Jo Lynne Rokita (rokita@chop.edu).

#### Materials availability

This study did not create new, unique reagents.

#### Data and code availability

Merged summary files for OpenPedCan v12 are openly accessible in [CAVATICA](https://cavatica.sbgenomics.com/u/cavatica/opentarget) or via download script in the [https://github.com/PediatricOpenTargets/OpenPedCan-analysis](https://github.com/PediatricOpenTargets/OpenPedCan-analysis) repository.
Cancer group summary data are visible within the NCI's pediatric [Molecular Targets Platform](https://moleculartargets.ccdi.cancer.gov/) and cohort, cancer group, and individual data are visible within [PedcBioPortal](https://pedcbioportal.kidsfirstdrc.org/study/summary?id=openpedcan_v12)

OpenPedCan analysis modules were developed within OpenPBTA [@doi:10.1016/j.xgen.2023.100340], modified based on OpenPBTA, or newly created and can be found within the following publicly available repositories.
OpenPBTA module analyses can be found at [https://github.com/AlexsLemonade/OpenPBTA-analysis](https://github.com/AlexsLemonade/OpenPBTA-analysis).
OpenPedCan module analyses can be found at [https://github.com/PediatricOpenTargets/OpenPedCan-analysis](https://github.com/PediatricOpenTargets/OpenPedCan-analysis).
OpenPedCan api code can be found at [https://github.com/PediatricOpenTargets/OpenPedCan-api](https://github.com/PediatricOpenTargets/OpenPedCan-api).

Software versions are documented in **Table XX**.

#### Data releases


### EXPERIMENTAL MODEL AND STUDY PARTICIPANT DETAILS
<!-- TODO: add description of all studies here -->



### METHOD DETAILS

#### Nucleic acids extraction and library preparation
<!-- TODO: add by study here, or refer to publication -->


#### Data generation
<!-- TODO: add by study here, or refer to publication -->


#### DNA WGS Alignment
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].

#### Quality Control of Sequencing Data
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].


##### SNP calling for B-allele Frequency (BAF) generation
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].


#### Somatic Mutation Calling

##### SNV and indel calling
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].


##### VCF annotation and MAF creation
<!-- TODO: update VEP version -->


##### Gather SNV and INDEL Hotspots
<!-- TODO: needs update -->


##### Consensus SNV Calling
<!-- TODO: needs update -->


#### Somatic Copy Number Variant Calling (WGS samples only)
<!-- TODO: needs update -->

##### Consensus CNV Calling
<!-- TODO: needs update -->


#### Somatic Structural Variant Calling (WGS samples only)
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].


#### Methylation Analysis
<!-- include QC, b-value/m-value calculations, packages, dkfz classification -->


#### Gene Expression

##### Abundance Estimation
<!-- TODO: needs update -->


##### Gene Expression Matrices with Unique HUGO Symbols
<!-- TODO: needs update, include liftover for TCGA/GTEX -->


##### Gene fusion detection
<!-- TODO: needs update -->


### QUANTIFICATION AND STATISTICAL ANALYSIS

##### Focal Copy Number Calling (`focal-cn-file-preparation` analysis module)



##### Gene Set Variation Analysis (`gene-set-enrichment-analysis` analysis module)
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].



##### Fusion prioritization (`fusion_filtering` analysis module)




#### Mutational Signatures (`mutational-signatures` analysis module)



### Tumor Mutation Burden (`snv-callers` analysis module)



#### Clinical Data Harmonization

##### WHO Classification of Disease Types


##### Molecular Subtyping
Here, we build upon the molecular subtyping performed in OpenPBTA [@doi:10.1016/j.xgen.2023.100340].

**High-grade gliomas.**
<!-- TODO: needs update - DHG, IHG -->

A new high-grade glioma entity called infant-type hemispheric gliomas (IHGs), characterized by distinct gene fusions enriched in receptor tyrosine kinase (RTK) genes including _ALK_, _NTRK1/2/3_, _ROS1_ or _MET_, was identified in 2021 [@doi: 10.1038/s41467-019-12187-5].
To identify IHG tumors, first, tumors which were classified as "IHG" by the DKFZ methylation classifier or diagnosed as "infant type hemispheric glioma" from `pathology_free_text_diagnosis` were selected [@doi:10.1038/nature26000]. 
Then, the corresponding tumor RNA-seq data were utilized to seek the evidence for RTK gene fusion. 
Based on the specific RTK gene fusion present in the samples, IHGs were further classified as "IHG, ALK-altered", "IHG, NTRK-altered", "IHG, ROS1-altered", or "IHG, MET-altered". 
If no fusion was observed, the samples were identified as "IHG, To be classified". 


**Atypical teratoid rhabdoid tumors.**

Atypical teratoid rhabdoid tumors (ATRT) tumors were categorized into three subtypes: "ATRT, MYC", "ATRT, SHH", and "ATRT, TYR" [@doi:10.1093/neuonc/noz235]. 
In OpenPedCan, the molecular subtyping of ATRT was based solely on the DNA methylation data. 
Briefly, ATRT samples with a high confidence DKFZ methylation subclass score (>= 0.8) were selected and subtypes were assigned based on the DKFZ methylation subclass [@doi10.1038/nature26000]. 
Samples with low confidence DKFZ methylation subclass scores (< 0.8) were identified as "ATRT, To be classified". 

**Neuroblastoma tumors.**

Neuroblastoma (NBL) tumors with a pathology diagnosis of neuroblastoma, ganglioneuroblastoma, or ganglioneuroma were subtyped based on their MYCN copy number status as either "NBL, MYCN amplified" or "NBL, MYCN non-amplified". 
If `pathology_free_text_diagnosis` was "NBL, MYCN non-amplified" and the genetic data suggested MYCN amplification, the samples were subtyped as "NBL, MYCN amplified". 
On the other hand, if `pathology_free_text_diagnosis` was "NBL, MYCN amplified" and the genetic data suggested MYCN non-amplification, the RNA-Seq gene expression level of _MYCN_ was used as a prediction indicator. 
In those cases, samples with _MYCN_ gene expression above or below the cutoff (TPM >= 140.83) were subtyped as "NBL, MYCN amplified" and "NBL, MYCN non-amplified", respectively. 
_MYCN_ gene expression was also used to subtype samples without DNA sequencing data. 
If a sample did not fit none of these situations, it was denoted as "NBL, To be classified".  



###### Integration of brain tumor methylation classifications 


#### TP53 Alteration Annotation (`tp53_nf1_score` analysis module)
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].


#### Prediction of participants' genetic sex
Please refer to the OpenPBTA manuscript for details [@doi:10.1016/j.xgen.2023.100340].


#### Selection of independent samples (`independent-samples` analysis module)



